src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/entity/Payment.java

```diff
@@ -0,0 +1,73 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.entity;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.time.Instant;
+import java.util.List;
+import java.util.UUID;
+
+@Entity
+@Table(name = "payments_table")
```

Comment:
I don't think we need a table called payments_table, it could likely be payments on its own (singular)